lib/ansible/modules/storage/netapp/na_ontap_software_update.py

```diff
@@ -76,11 +78,10 @@ class NetAppONTAPSoftwareUpdate(object):
     """
 
     def __init__(self):
-
         self.argument_spec = netapp_utils.na_ontap_host_argument_spec()
         self.argument_spec.update(dict(
             state=dict(required=False, type='str', choices=['present', 'absent'], default='present'),
-            node=dict(required=False, type='list'),
+            nodes=dict(required=False, type='list'),
```

Comment:
please add aliases for node.

This will allow older Ansible 2.7 Playbooks to still work.